src/client/components/ChannelPicker.tsx

```diff
@@ -76,20 +76,25 @@ export function ChannelPicker({
 
   const handleTextChange = useCallback(
     (e: React.ChangeEvent<HTMLInputElement>) => {
-      const text = e.target.value.trim();
+      const text = e.target.value;
```

Comment:
Channel names could not have spaces before so I trimmed it off.  Now they can since a DM may have a First Last Name